converters/toZigbee.js

```diff
@@ -586,6 +586,30 @@ const converters = {
             await entity.read('lightingColorCtrl', ['colorTemperature']);
         },
     },
+    light_colortemp_startup: {
+        key: ['color_temp_startup', 'color_temp_startup_percent'],
+        convertSet: async (entity, key, value, meta) => {
+            if (typeof value === 'string' && value.toLowerCase() == 'previous') {
+                // 0xffff = restore previous value
+                value = 65535;
+            } else if (key === 'color_temp_startup_percent') {
+                value = Number(value) * 3.46;
+                value = Math.round(value + 154).toString();
+            }
+
+            value = Number(value);
+
+            // ensure value within range
+            const [colorTempMin, colorTempMax] = light.findColorTempRange(entity, meta.logger);
+            value = light.clampColorTemp(value, colorTempMin, colorTempMax, meta.logger);
+
+            await entity.write('lightingColorCtrl', {startUpColorTemperature: value}, utils.getOptions(meta.mapped, entity));
+            return {state: {color_temp_startup: value}};
```

Comment:
@Koenkk: Is it OK to not return a state? I think perhaps a read call would be better.

That way bulbs that return UNSUPPORTED_ATTRIBUTE don't have it in there cached state.

Reply:
In case of an unsupported_attribute an exception should be thrown at `entity.write` so the return statement is never hit.